docs/apache-airflow/common-pitfall.rst

```diff
@@ -0,0 +1,202 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+Common Pitfalls
+===============
+
+Airflow Configurations
+^^^^^^^^^^^^^^^^^^^^^^
+
+Configuring parallelism
+-----------------------
+
+These configurations are executor agnostic.
+
+- :ref:`config:core__parallelism`
+
+  The maximum number of task instances that Airflow can run concurrently.
+  This number usually reflects the number of task instances with the
+  running state in the metadata database.
+
+- :ref:`config:core__dag_concurrency`
+
+  The maximum number of task instances to be allowed to run concurrently
+  per DAG. To calculate the number of tasks that is running concurrently
+  for a DAG, add up the number of running tasks for all DAG runs of the DAG.
+  This is configurable at the DAG level with ``concurrency``.
+
+- :ref:`config:core__max_active_runs_per_dag`
+
+  The maximum number of active DAG runs per DAG. The scheduler will not
+  create more DAG runs if it reaches the limit. This is configurable at
+  the DAG level with ``max_active_runs``.
```

Comment:
These appear to duplicate the content in [here](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html)
why not update configurations to be more descriptive and reference here?


There some content in [FAQ How can my airflow dag run faster](https://airflow.apache.org/docs/apache-airflow/stable/faq.html#how-can-my-airflow-dag-run-faster) that are correlated with this too, WDYT?

Reply:
Yeah that makes sense! I will update the configuration description and update that section with the links to those airflow configurations.